### DISCUSSION TITLE
What could be the reason for 6lo send queue heap up?

### DISCUSSION THREAD

**Discussion by lancersky** [openthread/openthread#7926] [Q&A] [answered]
**Describe the bug** A clear and concise description of what the bug is.

![0008624393e9aa71e366d3ebfa6262f](https://user-images.githubusercontent.com/11283852/179914161-ddf73453-9d38-462f-b294-487be3aa07f3.jpg)

![image](https://user-images.githubusercontent.com/11283852/179913412-446efef1-a531-4b02-8d00-0278b1445c8c.png)

![image](https://user-images.githubusercontent.com/11283852/179913205-74bf0285-514f-40f5-a67b-b9cf477dfbb0.png)

**To Reproduce** Information to reproduce the behavior, including:

1. Git commit id
https://github.com/openthread/ot-br-posix/commit/5be35949c6f1fd4cc3a8e63169cec6a2211135a4
RCP(EFR32MG21F768) firmware base on GSDK4.0.2.
3. IEEE 802.15.4 hardware platform
4. Build steps
5. Network topology

**Expected behavior** A clear and concise description of what you expected to happen.

**Console/log output** If applicable, add console/log output to help explain your problem.

**Additional context** Add any other context about the problem here.

**Comment by jwhui** [comment, marked as answer]
The `6lo send` queue not being serviced is typically an indicator that the radio driver is stuck. In particular, the radio was requested to send a packet, but the radio driver never provided indication that it completed its transmission operation.

See:

https://github.com/openthread/openthread/blob/424005f6957d57612ba555aeeab99fdd874c364a/src/core/mac/mac.cpp#L921

https://github.com/openthread/openthread/blob/424005f6957d57612ba555aeeab99fdd874c364a/src/core/mac/mac.cpp#L1259
